Add Alliant Credit Union

kishoresurana opened the pull request.

stephengroat commented on _data/banking.yml:
```diff
@@ -1,647 +1,654 @@
+      url: www.alliantcreditunion.com
+      twitter: AlliantCU
+      facebook: alliantcreditunion
+      img: actorsfcu.png
```
Please use the proper image

stephengroat reviewed (dismissed):
Please fix the image issue, `actorsfcu.png` is not the right image

Commit:
Add Alliant Credit Union

Signed-off-by: Carlgo11 <github@carlgo11.com>

Jawshy committed:
Banking: Alliant Credit Union: Supports HTTPS

'https://secure.alliantcreditunion.org/' also works. Must include 'https://' as it will not redirect.

Jawshy reviewed (approved):
`looks-good`
